contracts/contracts/mocks/EternalStorageMock.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.23;
```

Comment:
I can understand the usefulness of this in remix. But I might be a little nervous if we had tests relying on the mock rather than the actual contract. Since each test only has to be written once, I think it would probably be worth it to use the actual contract in the tests. Especially considering what's at stake.

Reply:
This isn't used in tests at all. It's only for Remix.

Reply:
I just removed one unneeded import of this file, which should make the lack of dependence on this clearer.